source/geod24/LocalRest.d

```diff
@@ -1501,6 +1522,12 @@ unittest
     auto node = RemoteAPI!API.spawn!Node();
     assert(node.sleepFor(80) == 42);  // no timeout
 
+    // custom timeout
+    node.ctrl.withTimeout(500.msecs,
+        (scope api) {
```

Comment:
If I make the delegate templated in `withDuration`, then I must specify the correct type here. `(scope RemoteAPI!API api)` rather than just `(scope api)`

Reply:
On the other hand, making it templated allows passing any callable type.